### PR TITLE
Update CCIP Read page

### PR DESCRIPTION
**Pull request by jlwllmr**
Amending text to clarify the distinction between Chainlink's CCIP and the CCIP Read function developed by ENS. 
To this end, adds a separate CCIP subheading.

Also simplifies the filename.

### CODE REVIEW

File: docs/get-started/tooling/cross-chain/ccip-read.mdx [hunk 1]
```diff
@@ -27,7 +26,17 @@ Read more about sparse Merkle trees in our [architecture documentation](../../co
...
+
+The Chainlink CCIP is middleware designed to enable traditional backends to securely interact with 
+blockchains, preventing the need for organisations to devote resources to developing case-by-case
+solutions to this problem. This enables applications to transfer assets and data between chains,
+allowing developers to access different chains for their relative strengths, and also access a 
+larger audience.
+
+Read more about CCIP in the [Chainlink documentation](https://docs.chain.link/ccip).
+
+## Linea ENS
```
Comment (mapachurro) [inline docs/get-started/tooling/cross-chain/ccip-read.mdx:39]: This is a weird warning / error. I mean, I get it, but we should be allowed to have initialisms in titles :sob:

Comment (mapachurro) [reply docs/get-started/tooling/cross-chain/ccip-read.mdx:39]: Any way we can escape Vale here?

**Review by mapachurro — APPROVED** [review]
Builds Good, Looks Good

image

If we merge with the Vale failure, does it break?

File: docs/get-started/tooling/cross-chain/ccip-read.mdx (hunk 1 above)
Comment (jlwllmr) [reply docs/get-started/tooling/cross-chain/ccip-read.mdx:39]: There may be a way to for a specific line, but I'm not aware of it.

I made a note to add `ENS` to accepted acronyms/initialisms in Vale the next time I submit a PR!

**Comment by jlwllmr** [comment]
@mapachurro no — Vale errors don't affect the build!

**Pull request merged**